package.json

```diff
@@ -24,12 +24,13 @@
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
     "compile:script": "tsc -p script",
-    "lint": "yarn compile:tslint && yarn compile:script && yarn tslint && yarn eslint-check && yarn eslint",
-    "lint:fix": "yarn compile:tslint && yarn tslint:fix && yarn eslint:fix",
+    "lint": "yarn lint:src && yarn lint:prettier || (echo '\u001b[1m\u001b[32m→ To fix some of these errors, run \u001b[39m\u001b[22m\u001b[1m\u001b[32m\u001b[4myarn lint:fix\u001b[24m\u001b[39m\u001b[22m' && false)",
+    "lint:fix": "yarn lint:src:fix && yarn lint:prettier --write",
+    "lint:prettier": "prettier '**/*.md' '**/*.scss' --list-different",
```

Comment:
Drop the quote marks from these paths so they get expanded by the shell. Just tested this works in Git Bash, Git CMD and PowerShell (all the Windows shells I can think of, basically):

<img width="1333" src="https://user-images.githubusercontent.com/359239/34668918-c72421ca-f445-11e7-9eed-5061bb8234a4.png">

Reply:
Does `**/*.{md,scss}` work, too?

Reply:
Seems fine here:

<img width="1255" src="https://user-images.githubusercontent.com/359239/34669249-29130a76-f447-11e7-8d44-af76d5f11b43.png">